tests/unit/samplegen/test_template.py

```diff
@@ -153,9 +153,9 @@ def test_render_print_args():
     check_template(
         '''
         {% import "feature_fragments.j2" as frags %}
-        {{ frags.renderPrint(["Molluscs: %s, %s, %s", "squid", "clam", "whelk"]) }}
+        {{ frags.renderPrint(["$resp %s %s", "$resp.squids", "$resp.clams"]) }}
         ''',
-        '\nprint("Molluscs: {}, {}, {}", squid, clam, whelk)\n'
+        '\nprint("$resp {} {}", response.squids, response.clams)\n'
```

Comment:
nitty nit nit suggestion: Consider having the expected string also be specified using `'''`: this will make it more readable and take care of the `\n`s

Reply:
Yeah, bad call on my part. My initial judgement was that using triple-quoted strings for one-liners was less readable, but the alternative is not great.